### PR TITLE
Homework_02(finally complete)

### PR DESCRIPTION
**Pull request by NikitaNazarov1**

### CODE REVIEW

File: HW_02/Nazarov_nikita/nikita_nazarov_hw2_t1.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+puts 'The hour hand goes way in x degrees in 15 minutes'
+puts 'so...'
+x = 7.5
+puts 'the angle between the hours and the minutes hands is'
+puts x
```
Comment (labs-ruby) [inline HW_02/Nazarov_nikita/nikita_nazarov_hw2_t1.rb]: why double files?

Comment (labs-ruby) [reply HW_02/Nazarov_nikita/nikita_nazarov_hw2_t1.rb]: nikita_nazarov_hw2_t1.rb and nikita_nazarov_hw_02_t_01.rb?

File: HW_02/Nazarov_nikita/nikita_nazarov_hw_02_t_02.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+puts 'Try 1/3+4*6'
+puts 'so...'
+x = 1 / 3 + 4 * 6
+puts 'x is:'
+puts x
+puts 'Yeah, success!'
```
Comment (labs-ruby) [inline HW_02/Nazarov_nikita/nikita_nazarov_hw_02_t_02.rb]: 0,333333 - just ignore?

File: HW_02/Nazarov_nikita/nikita_nazarov_hw2_t1.rb (hunk 1 above)
Comment (NikitaNazarov1) [reply HW_02/Nazarov_nikita/nikita_nazarov_hw2_t1.rb]: fix

**Pull request merged**